node/env0-deploy-flow.js

```diff
@@ -0,0 +1,62 @@
+const DeployUtils = require('./env0-deploy-utils');
+
+const deployUtils = new DeployUtils();
+
+const runDeployment = async (options) => {
+    await DeployUtils.init(options);
+    if (options.action === 'Deploy') {
+        await createAndDeploy(options);
+    }
+    else if (options.action === 'Destroy') {
+        await destroy(options);
+    }
+    else {
+        throw new Error(`Action ${options.action} is Invalid, Valid actions are Deploy and Destroy`);
+    }
+};
+
+const createAndDeploy = async (options) => {
+    console.log('Starting deployment');
+    let environment = await deployUtils.getEnvironment(options.environmentName, options.organizationId);
+    console.log(`got the following environment: ${JSON.stringify(environment)}`);
+
+    if (!environment) {
+        console.log('did not find an environment');
+        environment = await deployUtils.createEnvironment(options.environmentName, options.organizationId, options.projectId);
+    }
+    await setConfigurationFromOptions(options, environment);
+    await deployUtils.pollEnvironmentStatus(environment.id);
+    await deployUtils.deployEnvironment(environment, options.revision, options.blueprintId);
+    await deployUtils.pollEnvironmentStatus(environment.id);
+};
+
+const destroy = async (options) => {
+    console.log('Starting destroying an environment');
+    const environment = await deployUtils.getEnvironment(options.environmentName, options.organizationId);
+    console.log(`got the following environment: ${JSON.stringify(environment)}`);
```

Comment:
`deployUtils.getEnvironment` already logs this, can be removed

Reply:
Done.